src/Microsoft.ML.OnnxTransformer/OnnxCatalog.cs

```diff
@@ -63,14 +63,23 @@ public static class OnnxCatalog
         => new OnnxScoringEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnNames, inputColumnNames, modelFile, gpuDeviceId, fallbackToCpu);
 
         /// <summary>
-        /// Creates a new instance of <see cref="DnnImageFeaturizerEstimator"/> which applies a pre-trained DNN model to featurize an image.
+        /// Creates a new instance of <see cref="DnnImageFeaturizerEstimator"/> which applies one of the pre-trained DNN models in
+        /// <see cref="DnnImageModelSelector"/>to featurize an image.
         /// </summary>
         /// <param name="catalog">The transform's catalog.</param>
         /// <param name="outputColumnName">The name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
         /// <param name="modelFactory">An extension method on the <see cref="DnnImageModelSelector"/> that creates a chain of two
         /// <see cref="OnnxScoringEstimator"/> (one for preprocessing and one with a pretrained image DNN) with specific models
         /// included in a package together with that extension method.</param>
-        /// <param name="inputColumnName">Name of column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="inputColumnName">Name of column to transform.
+        /// If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[StochasticGradientDescent](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/ImageAnalytics/DnnFeaturizeImage.cs)]
```

Comment:
>ImageAnalytics [](start = 113, length = 14)

I think ImageAnalytics should go under Transforms. The hierarchy under Microsoft.ML.Samples/Dynamic should mirror the catalog hierarchy as much as possible.

Reply:
thanks for moving the files. please fix this sample path and the path for the other transforms that got moved. 

---
In reply to: [261433488](https://github.com/dotnet/machinelearning/pull/2799#discussion_r261433488) [](ancestors = 261433488)